docs-source/content/userguide/managing-operators/using-the-operator/the-rest-api.md

```diff
@@ -14,7 +14,12 @@ You can access most of the REST services using `GET`, for example:
 * To obtain a list of domains, send a `GET` request to the URL `/operator/latest/domains`
 * To obtain a list of clusters in a domain, send a `GET` request to the URL `/operator/latest/domains/<domainUID>/clusters`
 
-All of the REST services require authentication.  Callers must pass in a valid token header and a CA certificate file.  Callers should pass in the `Accept:/application/json` header.
+All of the REST services require authentication.  Callers must pass in a valid token header and a CA certificate file.  In previous operator versions, the operator would perform authentication and authorization checks using the Kubernetes token review and subject access review API's and then update the Domain resource using the operator's privileges.  Now the operator will, by default, use the Caller's bearer token to perform the underlying update to the Domain resource using the Caller's privileges and thus delegating authentication and authorization checks directly to the Kubernetes API Server (see [REST interface configuration]({{< relref "/userguide/managing-operators/using-the-operator/using-helm.md#rest-interface-configuration" >}})).  
```

Comment:
operator would perform ->operator performed
API's -> APIs, (no apostrophe, add a comma)
and then update -> and then updated
Now the operator will, by default, use -> Now, by default, the operator will use
Caller's bearer token -> caller's bearer token 
Caller's privileges -> caller's privileges